com.regnosys.rosetta.tests/src/com/regnosys/rosetta/generator/java/rule/ConditionTestHelper.xtend

```diff
@@ -8,20 +8,47 @@ import com.rosetta.model.lib.path.RosettaPath
 import java.util.Map
 import java.util.Optional
 import com.regnosys.rosetta.generator.java.RosettaJavaPackages.RootPackage
+import com.google.inject.Injector
+import com.google.inject.Guice
+import com.google.inject.AbstractModule
+import com.rosetta.model.lib.functions.ConditionValidator
+import com.rosetta.model.lib.functions.DefaultConditionValidator
+import com.rosetta.model.lib.functions.ModelObjectValidator
+import com.rosetta.model.lib.functions.NoOpModelObjectValidator
 
 class ConditionTestHelper {
 	
 	@Inject extension ModelHelper
 	@Inject extension RosettaExtensions
 	@Inject extension CodeGeneratorTestHelper
 	
+	final Injector injector
```

Comment:
Currently, the test code responsible for running conditions didn't inject its dependencies like functions that the condition needs to call. Fixed here.